app/api/v2/events.py

```diff
@@ -168,6 +168,7 @@ def publish_handler(
 ):
     """Handler for ATBD Publication"""
     [version] = atbd.versions
+    print(f"THIS ENTITY SHOULD BE PROCESSED: {[version]}")
```

Comment:
Yea, I did leave this on purpose. There's no logs that get surfaced in the stack so my hope is this comment this will at least surface some PDF generation issues if they come up in the future.

Reply:
[UGA latex doc](https://www.overleaf.com/latex/templates/agu-geophysical-research-letters-agutex-article/rnyzczmyvkbj)